src/querl.hrl

```diff
@@ -0,0 +1 @@
+-type already_present_error() :: {error, {already_present, Key :: any()}}.
```

Comment:
I'd rather inline this where used than making it difficult to read specs in the source if you don't mind :)